settings/timesheets.sublime-settings

```diff
@@ -5,5 +5,19 @@
 
     // Jira ticket URL, e.g.
     // "https://jira.example.com/browse/{}"
-    "jira_ticket_url": "https://jira.iponweb.net/browse/{}"
-}
\ No newline at end of file
+    "jira_ticket_url": "https://jira.iponweb.net/browse/{}",
+
+    // JIRA endpoint that will be called to gather completion candidates
+    // Feel free to adjust the query and the number of results here.
+    "jira_completion_url": "https://jira.iponweb.net/rest/api/2/search?jql=(labels+in+(nbt,sft)+OR+watcher+in+({username})+OR+assignee+in+({username})+OR+reporter+in+({username}))+ORDER+BY+updated+DESC,+created+DESC&maxResults=100&fields=id,key,summary",
+
+    // Filename where to put internal messages. If not set or empty,
+    // messages are not logged.
+    //"log_filename": "/tmp/sublime_jira_completion.log",
+
+    // Trigger that activates the completion.
+    "trigger": "JI",
```

Comment:
I suggest to add prefix for these new settings, so it's clear they belong to Jira config. E.g. "jira_autocomplete_trigger", "jira_timeout", "jira_log_filename".